### PR TITLE
removed package.json.lock rebuilt with yarn i.e. yarn.lock

### PR DESCRIPTION
**Pull request by QARTHUR-NIB**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# development
+NEXT_PUBLIC_API_URL="http://localhost:1337/api"
```
Comment (Stanico-Knowles) [inline .env:2]: .env pushed up to repo? wilddddd

File: .env.me [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#    This file uniquely authorizes you against this project in dotenv-vault.    #
+#                 Do NOT commit this file to source control.                    #
+#                                                                               #
+#                  Generated with 'npx dotenv-vault login'                      #
+#                                                                               #
+#                  Learn more at https://dotenv.org/env-me                      #
+#                                                                               #
+#################################################################################
+
+DOTENV_ME=me_5f84b876b536ec50d0f7d8f404ef3cb43c7b81dc21d26bf35779c49c1650f788
```
Comment (Stanico-Knowles) [inline .env.me:12]: should also be in gitignore

**Review by Stanico-Knowles — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (QARTHUR-NIB) [reply .env:2]: lmfao bey who set up the gitignore?????

Comment (Stanico-Knowles) [reply .env:2]: NextJS

Comment (QARTHUR-NIB) [reply .env]: Somehow it didn't have an entry for .env files in there. I've taken the liberty of adding it oh great senior SIR! i apologize for this inconvenience and hope to be half the developer you are one day @Stanico-Knowles

**Pull request merged**